tox.ini

```diff
@@ -49,7 +49,9 @@ commands =
               --skip {toxinidir}/.mypy_cache \
               --skip {toxinidir}/.ruff_cache \
               --skip {toxinidir}/icon.svg \
-              --skip {toxinidir}/src/grafana_dashboards
+              --skip {toxinidir}/src/grafana_dashboards \
+              --skip {toxinidir}/tox.ini \
+              -L aNULL,anull
```

Comment:
What's that?

Reply:
This prevents linting error due to:
https://github.com/canonical/mimir-coordinator-k8s-operator/pull/20/files#diff-eab582b474ff3c4ac3d1aba17cc59c806e8dc266e6c48a0ed35d3a60edbcc9d5R341